docker/standalone.xml

```diff
@@ -385,6 +385,8 @@
             <filters>
                 <response-header name="server-header" header-name="Server" header-value="WildFly/10"/>
                 <response-header name="x-powered-by-header" header-name="X-Powered-By" header-value="Undertow/1"/>
+
+                <filter name="request-dumper" module="io.undertow.core" class-name="io.undertow.server.handlers.RequestDumpingHandler"/>
```

Comment:
If it works it does dump the Respose too https://github.com/undertow-io/undertow/blob/master/core/src/main/java/io/undertow/server/handlers/RequestDumpingHandler.java#L118